MusiciansAPP/MusiciansAPP.Domain/Track.cs

```diff
@@ -12,14 +12,23 @@ public class Track
     [MaxLength(TrackConstraints.NameMaxLength)]
     public string Name { get; set; }
 
-    [Required]
     [Range(TrackConstraints.PlayCountMinValue, int.MaxValue)]
-    public int PlayCount { get; set; }
+    public int? PlayCount { get; set; }
 
     [Range(TrackConstraints.DurationInSecondsMinValue, int.MaxValue)]
     public int? DurationInSeconds { get; set; }
     public Guid ArtistId { get; set; }
```

Comment:
Why do we need this field? EF can bind an object using the object itself.
Let's avoid using it.